apps/back/src/Entity/User.php

```diff
@@ -28,14 +30,27 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
     private string $password;
 
     #[ORM\Column(type: 'boolean')]
-    private $isVerified = false;
+    private bool $isVerified = false;
+
+    #[ORM\Column(type: "datetime")]
+    private \DateTimeImmutable $registerAt;
+    #[ORM\Column(type: 'integer')]
```

Comment:
FYI, on future projects, you won't see mapping through Annotation (or attribute) because of the coupling it implies with Doctrine

Do not change it now, though.